src/MongoDB/WriteResult.c

```diff
@@ -31,10 +31,17 @@
 #include "MongoDB/WriteError.h"
 #include "WriteResult_arginfo.h"
 
+#define PHONGO_WRITERESULT_CHECK_ACKNOWLEDGED(method)                                                                                                                                         \
+	if (!mongoc_write_concern_is_acknowledged(intern->write_concern)) {                                                                                                                       \
+		php_error_docref(NULL, E_DEPRECATED, "Calling MongoDB\\Driver\\WriteResult::" method "() for an unacknowledged write is deprecated and will throw an exception in ext-mongodb 2.0."); \
```

Comment:
I'd suggest omitting the trailing period here. IIRC, we only do that when a message includes multiple sentences.

Reply:
Removed the trailing period, which hopefully also fixes tests.